src/commands.cpp

```diff
@@ -4731,6 +4731,28 @@ void CommandChatPresenceURL::procresult()
     }
 }
 
+CommandRegisterPushNotification::CommandRegisterPushNotification(MegaClient *client, int deviceType, const char *token)
+{
+    this->client = client;
+    cmd("spt");
+    arg("p", deviceType);
+    arg("t", token);
+    notself(client);
```

Comment:
Is it needed a `notself` here?
I mean, does this command generate an any action packet?